Kconfig

```diff
@@ -106,6 +106,19 @@ config KAS_INCLUDE_LXDE
 	default "kas/opt/lxde.yml"
 	depends on LXDE
 
+config EIO
+	bool "EIO subsystem support"
+	help
```

Comment:
Not 'default y'? But we will enable it by default in the next example image release, I assume.

Reply:
Yes, we will enable it in our example image release building (internal CI), which will be uploaded into SIOS.

Reply:
Then I would suggest to turn it on in the selection here by default so that users can easily reproduce what they also get prebuilt, just like for the Node-RED things.

Reply:
But that will leave the user a building error that the EIO binaries do not exist...

Reply:
Unless some dummy files are used for building, and print a warning to user that dummy files are used, and non interested users can just ignore that one.

Reply:
Hmpf, just realized that as well. So this whole thing is also untested in CI? I suppose we can't easily fetch the missing files from their new homes, can we?

Reply:
> Hmpf, just realized that as well. So this whole thing is also untested in CI? I suppose we can't easily fetch the missing files from their new homes, can we?

We will copy the files into some folder at the prepare stage in CI.

No it's not possible to auto fetch the files, thanks to SIOS's policy that login before download.

Reply:
I'm not seeing any changes to this CI (downstream is irrelevant for QA).

Reply:
Can't this be accessed via an API key? We could store that API key in a CI secret. It's ridiculous that the automation is blocked because of this non-sense.

Reply:
I've checked on the SIOS website, and no API key or token could be found... Maybe I am missing something...

Reply:
Change to stub the EIO binaries in the CI. And now CI will build EIO for all of the example image variants.